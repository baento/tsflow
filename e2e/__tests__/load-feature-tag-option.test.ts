import runJest from "../runJest";
import { extractSummary } from "../utils";

test("loadFeature tag options", () => {
  const result = runJest("load-feature-tag-option");

  const { rest } = extractSummary(result.stderr);

  expect(rest).toMatchSnapshot();
});
