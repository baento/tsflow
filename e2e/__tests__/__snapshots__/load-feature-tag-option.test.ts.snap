// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadFeature tag options 1`] = `
"PASS __tests__/index.test.ts
  loadFeature with tag option - Scenario
    ✓ Add two number
    ✓ Add two number
    ✓ Add two number
  loadFeature with tag option - Scenario Outline
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)
    ✓ Add two number (with Examples)"
`;
